ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -109,29 +123,48 @@ function ImagePage() {
             </PageSection>
         );
     } else {
+        const SBOMButtonWrapper = isScannerV4Enabled ? React.Fragment : ScannerV4RequiredTooltip;
         const sha = imageData?.id;
         mainContent = (
             <>
                 <PageSection variant="light">
                     {imageData ? (
                         <Flex
                             direction={{ default: 'column' }}
-                            alignItems={{ default: 'alignItemsFlexStart' }}
+                            alignItems={{ default: 'alignItemsStretch' }}
                         >
-                            <Title headingLevel="h1" className="pf-v5-u-m-0">
-                                {imageDisplayName}
-                            </Title>
-                            {sha && (
-                                <ClipboardCopy
-                                    hoverTip="Copy SHA"
-                                    clickTip="Copied!"
-                                    variant="inline-compact"
-                                    className="pf-v5-u-display-inline-flex pf-v5-u-align-items-center pf-v5-u-mt-sm pf-v5-u-mb-md pf-v5-u-font-size-sm"
+                            <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+                                <Flex
+                                    direction={{ default: 'column' }}
+                                    spaceItems={{ default: 'spaceItemsSm' }}
                                 >
-                                    {sha}
-                                </ClipboardCopy>
-                            )}
-                            <ImageDetailBadges imageData={imageData} />
+                                    <Title headingLevel="h1">{imageDisplayName}</Title>
+                                    {sha && (
+                                        <ClipboardCopy
+                                            hoverTip="Copy SHA"
+                                            clickTip="Copied!"
+                                            variant="inline-compact"
+                                            className="pf-v5-u-font-size-sm"
+                                        >
+                                            {sha}
+                                        </ClipboardCopy>
+                                    )}
+                                    <ImageDetailBadges imageData={imageData} />
+                                </Flex>
+                                {hasGenerateSBOMAbility && (
+                                    <FlexItem alignSelf={{ default: 'alignSelfCenter' }}>
+                                        <SBOMButtonWrapper>
+                                            <Button
+                                                variant="secondary"
+                                                onClick={() => {}}
+                                                isAriaDisabled={!isScannerV4Enabled}
```

Comment:
`isAriaDisabled` applies the disabled styles and prevents click events from firing, similar to the `isDisabled` prop. The difference is that the aria variant allows the element to be focused, both via hovering and keyboard navigation. This is needed in order to display the tooltip that describes to the user _why_ this button is disabled.